treeherder/model/models.py

```diff
@@ -210,14 +210,14 @@ def __str__(self):
 
 @python_2_unicode_compatible
 class Bugscache(models.Model):
-    id = models.AutoField(primary_key=True)
-    status = models.CharField(max_length=64, blank=True, db_index=True)
+    id = models.PositiveIntegerField(primary_key=True)
```

Comment:
It creates an auto-increment field instead:
https://docs.djangoproject.com/en/1.8/topics/db/models/#automatic-primary-key-fields

 `id` here refers to the bug id, which does not want to be auto_increment (and isn't on production, this was an inconsistency during the switch from refdata template SQL to using the ORM to generate the tables).